src/main/java/frc/robot/Constants.java

```diff
@@ -71,18 +81,62 @@
             public static final int kSpinnerMotorPort = 9; // motor for the wheel that spins the control panal
 
             public static final int kColorSensorPort = 0; // Color sensor that senses the colors on the control panel
+
+            /*values the color sensor values to detect the colors on control panel
+            red*/
+            public static final double kRedTargetR = 0.476;
+            public static final double kRedTargetG = 0.376;
+            public static final double kRedTargetB = 0.15;
+            //Yellow
+            public static final double kYellowTargetR = 0.381;
+            public static final double kYellowTargetG = 0.545;
+            public static final double kYellowTargetB = 0.136;
+            //Green
+            public static final double kGreenTargetR = 0.18;
+            public static final double kGreenTargetG = 0.568;
+            public static final double kGreenTargetB = 0.249;
+            //Blue
```

Comment:
Add a space before comments to make it easier to read.
```suggestion
            // Blue
```